### ISSUE TITLE
Doesn't work on Ubuntu 20.04

### ISSUE THREAD

**Issue by ftharyanto** [tzunghaor/inkscape-eps-export#9]
**Describe the bug**
When I tried to save it using EPS AI compatible it produced an error like this:
![image](https://user-images.githubusercontent.com/17604130/80682424-71461a00-8afd-11ea-9d11-164337e50d95.png)
I tried it on Ubuntu 20.04. However, when I tried it using Windows 10, it succeed with no error.

**Files**
Can't produce the EPS file, so I'll upload the SVG instead
[test.zip](https://github.com/tzunghaor/inkscape-eps-export/files/4556703/test.zip)

**To Reproduce**
1. Open Inkscape
2. Make something in the workspace
3. Save using EPS AI Compatible

**Expected behavior**
Successfully saved EPS AI Compatible with no error

**Comment by hervyqa** [comment]
are you install ghostcript?

**Comment by tzunghaor** [comment]
Thank you for the detailed bug report. Unfortunately I cannot try it out currently on Ubuntu 20.04  
Usually when there is an error in the export script, inkscape shows a more detailed error.

* Have you find a solution since?
* Can you save other formats in the same directory?

@hervyqa: my script doesn't use ghostscript

**Comment by ftharyanto** [comment]
> en there is an error in

Unfortunately it didn't show any detailed information about the error.

I just tried the extension using inkscape 1.0 on the same OS (Ubuntu 20.04) the extension runs smoothly with no error.

I don't know if this could be the root of the problem, but I installed Inkscape 1.0 with snap, while Inkscape 0.92.5 with Ubuntu repository (see image below)
![image](https://user-images.githubusercontent.com/17604130/81927199-36c09f00-961e-11ea-8929-2906d2ee6709.png)

I suspect that the Inkscape in Ubuntu repository is missing some dependencies (I mean it's only ~20MB compressed, ~130 MB uncompressed)

FYI: I kind of experenced the same with VLC. I tried converting video using VLC which was installed from Ubuntu repo, but when I tried to install VLC from snap, it could convert the video successfully.



> are you install ghostcript?

I have ghostscript pre-installed on my system (Ubuntu 20.04)

**Comment by ftharyanto** [comment]
> en there is an error in

Unfortunately it didn't show any detailed information about the error.

I just tried the extension using inkscape 1.0 on the same OS (Ubuntu 20.04) the extension runs smoothly with no error.

I don't know if this could be the root of the problem, but I installed Inkscape 1.0 with snap, while Inkscape 0.92.5 with Ubuntu repository (see image below)
![image](https://user-images.githubusercontent.com/17604130/81927199-36c09f00-961e-11ea-8929-2906d2ee6709.png)

I suspect that the Inkscape in Ubuntu repository is missing some dependencies (I mean it's only ~20MB compressed, ~130 MB uncompressed)

FYI: I kind of experenced the same with VLC. I tried converting video using VLC which was installed from Ubuntu repo, but when I tried to install VLC from snap, it could convert the video successfully.
